app/dashboard/models.py

```diff
@@ -1430,6 +1432,7 @@ class BountyFulfillment(SuperModel):
         ('FILECOIN', 'FILECOIN'),
         ('RSK', 'RSK'),
         ('XINFIN', 'XINFIN'),
+        ('ALGORANT', 'ALGORAND'),
```

Comment:
typo -> `ALGORAND` not `ALGORANT`

ps: redo migration?

Reply:
Doneeee. dankeee